test/jb_eth_payment_terminal_store/record_payment_from.test.js

```diff
@@ -0,0 +1,537 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+
+import { packFundingCycleMetadata } from '../helpers/utils';
+
+import jbController from '../../artifacts/contracts/JBController.sol/JBController.json';
+import jbDirectory from '../../artifacts/contracts/JBDirectory.sol/JBDirectory.json';
+import jBFundingCycleStore from '../../artifacts/contracts/JBFundingCycleStore.sol/JBFundingCycleStore.json';
+import jbFundingCycleDataSource from '../../artifacts/contracts/mocks/JBFundingCycleDataSourceMock.sol/JBFundingCycleDataSourceMock.json';
+import jbPayDelegate from '../../artifacts/contracts/mocks/JBPayDelegateMock.sol/JBPayDelegateMock.json';
+import jbPrices from '../../artifacts/contracts/JBPrices.sol/JBPrices.json';
+import jbProjects from '../../artifacts/contracts/JBProjects.sol/JBProjects.json';
+import jbTokenStore from '../../artifacts/contracts/JBTokenStore.sol/JBTokenStore.json';
```

Comment:
We should be able to use ../../artifacts/contracts/interfaces/... for all of these.